setup.cfg

```diff
@@ -22,9 +23,99 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+[metadata]
+name = invenio-github
+version = attr: invenio_github.__version__
+description = "Invenio module that adds GitHub integration to the platform."
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio github
+license = GPLv2
+author = CERN
+author_email = info@inveniosoftware.org
+platforms = any
+url = https://github.com/inveniosoftware/invenio-github
+classifiers =
+    Development Status :: 5 - Production/Stable
 
-[aliases]
-test=pytest
+[options]
+packages = find:
+zip_safe = False
+include_package_data = True
+python_requires = >= 3.7
+install_requires =
```

Comment:
many dependencies might not be needed, I just bumped what we had and removed the most obvious ones. 

e.g. `mistune` is not used, etc. I can have a look afterwards, for not it's not blocking